pull/src/handler.cc

```diff
@@ -113,8 +113,7 @@ static std::size_t WriteResponse(struct mg_connection* conn,
   return body.size();
 }
 
-bool MetricsHandler::handleGet(CivetServer* server,
-                               struct mg_connection* conn) {
```

Comment:
This change is unnecessary I think.

Reply:
I enabled warnings. The warning complains about an unused parameter. server is not used in the body.

Reply:
Use `ignore` instead of removing it directly.